static/scss/components/filter-bar.scss

```diff
@@ -60,6 +60,7 @@
         position: relative;
         z-index: 0;
         transition: padding-right .2s ease-out;
+        width: 400px;
```

Comment:
I recommend using a [placeholder value](https://protocol.mozilla.org/fundamentals/tokens.html) here. Closest I see is `$content-sm`. 

```suggestion
        width: $content-sm;
```

Preview at that length: 
![image](https://user-images.githubusercontent.com/2692333/137783209-0f9c3bcd-353d-41b0-874a-656762db7ce0.png)